spec/acceptance/api/cms_partial_spec.rb

```diff
@@ -11,26 +11,28 @@
       system_name: 'some-partial',
       liquid_enabled: false,
       handler: 'markdown'
-    )
+    ).extend(CMS::PartialRepresenter)
```

Comment:
It's this whole representers architecture... It overwrites the `to_json` method of an object to what is defined in the representer file.
If there's no `extend`, `to_json` is not overwritten.

Reply:
Because I was blind by the time I updated this test and left it there from a debug. I just removed the extend and it works as expected 🤦 
Updated in 2f382b0.

Reply:
Too bad